metron-platform/metron-integration-test/src/main/config/zookeeper/global.json

```diff
@@ -7,10 +7,12 @@
   "solr.collection": "metron",
   "solr.numShards": 1,
   "solr.replicationFactor": 1,
+  "flushIntervalInMs": "20000",
+  "flushOnTimeout": "false",
```

Comment:
What would you do if the flush flag were set to true and the flushIntervalInMs flag is unset?